.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,33 @@
+# Go to https://github.com/actions/stale for the full documentation of these options.
+
+name: 'Stale bot'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3.0.18
+        with:
+          # Rate limit per run, defaults to 30.
+          operations-per-run: 30
```

Comment:
I'm being explicit here with the rate limit, but we can also just rely on the implicit default of 30.